cloud-tenant-cd/pom.xml

```diff
@@ -43,6 +43,12 @@
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Consider manually adding the required dependencies instead of relying on `container-dev`.

Reply:
I disagree. container-dev is a convenience to ensure that you get imports for all packages that are provided from the container. It's better to keep the maintenance of that list in container-dev than spreading it around. By adding deps manually, you'll also have to add artifacts that you don't use directly to ensure that you get all necessary import-packages.